10/collection-meta.yaml

```diff
@@ -28,9 +28,7 @@ collections:
     repository: https://github.com/ISIB-Group/inspur.sm
   kubernetes.core:
     maintainers:
-      - jillr
-      - akasurde
-      - gravesm
+      - ansible-collections/cloud
```

Comment:
Can you adjust this similar to https://github.com/ansible-community/ansible-build-data/pull/333/files ? Then we can finally merge this.